pages/api/id/[form]/bearer.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { getSession } from "next-auth/client";
+import * as Constants from "../../../../lib/constant";
+import { executeQuery } from "../../../../lib/integration/queryManager";
+
+export const retrieve = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  try {
+    const session = await getSession({ req });
```

Comment:
Good opportunity for a middleware function here. See James's code https://github.com/cds-snc/platform-forms-client/blob/main/lib/middleware/httpRequestAllowed.ts

Reply:
I've used hooked James function. But you can already see that i kept the session validation inside my handler. The reason is that the isRequestAllow isn't generic enough to cover this case. So i wonder if you wanted to created a new middleware for this Endpoint as well. There's a way to make a generic middleware for this purpose but it might be out of this PR scope. 
ps:  Also updated James code  ( a starting point).

Reply:
I just meant write a simple middleware to test if a session exists. Not related to whether the method is allowed. No worries at this point its an extra thing.

Reply:
Done!